config/crd/bases/toolchain.dev.openshift.com_idlers.yaml

```diff
@@ -118,9 +118,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
I wonder why all the `status` have been removed by `controller-gen` 🤔

Reply:
@xcoulon yes, i was thinking the same. For reference - https://github.com/kubernetes-sigs/controller-tools/pull/630 - explains a bit more about the choice of removing status.